### PR TITLE
New achievements and achievements icons

### PR DESCRIPTION
**Pull request by ghost**
:cl: floyd and fikou for icons
 add:Adds an achievement regarding our ERP rules
add:Adds an achievement for getting through a veeeery long shift.
add:Adds an achievement for being the best miner
add:Adds icons for most achievements
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
please don't include bulletpoints inside the changelog entries

**Comment by Qustinnus** [comment]
my bad lol used to version control from work

**Comment by Fikou** [comment]
wtf im not credited im suing

**Comment by Qustinnus** [comment]
ya u are u must be blind smh

File: code/__HELPERS/roundend.dm [hunk 1]
```diff
@@ -188,10 +188,16 @@
...
+	var/speed_round = FALSE
+	if(world.time - SSticker.round_start_time <= 300 SECONDS)
+		speed_round = TRUE
+
 	for(var/client/C in GLOB.clients)
 		if(!C.credits)
 			C.RollCredits()
 		C.playtitlemusic(40)
+		if(speed_round)
+			C.give_award(/datum/award/achievement/misc/speed_round, C.mob)
```
Comment (ATH1909) [inline code/__HELPERS/roundend.dm]: maybe only give this achievement to someone if they were an antag that round (or alive and within visual range of an antag but not an antag themselves, to discourage suicide wizarding)?

File: code/datums/achievements/misc_achievements.dm [hunk 2]
```diff
@@ -75,3 +83,15 @@
 	name = "One Lean, Mean, Cleaning Machine"
```
Comment (ATH1909) [inline code/datums/achievements/misc_achievements.dm]: question: why are some of these achievement names capitalized, but not others?

Comment (Qustinnus) [reply code/datums/achievements/misc_achievements.dm]: because someone else added them

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (Qustinnus) [reply code/__HELPERS/roundend.dm]: whats a quick way to check antag status?

**Comment by Iatots** [comment]
Hey out of scope but the other day I tried self surgery and kept stabbing myself with a knife, unlocking the "how do i switch hands" cheevo multiple times in short succession. Can you give that a quick check here so I don't have to make an issue and shit?

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (ATH1909) [reply code/__HELPERS/roundend.dm]: Uh, I dunno. Maybe check the code for the roundend report screen, since that lists the round's antags?

Comment (MrDoomBringer) [reply code/__HELPERS/roundend.dm]: better yet, just give this achievement to people who haven't suicided?

**Comment by Timonkeyn** [comment]
can you explain how to get the new achievments?

**Comment by Qustinnus** [comment]
no that ruins the fun

File: code/__HELPERS/roundend.dm (hunk 1 above)
Comment (Qustinnus) [reply code/__HELPERS/roundend.dm]: how do you check if a client sudoku'd themselves

Comment (ExcessiveUseOfCobblestone) [reply code/__HELPERS/roundend.dm]: you can't (as of now) since ghost roles will remove the var.

Comment (actioninja) [reply code/__HELPERS/roundend.dm]: this is fine as is imo, achievements don't have to be things that are hard to do.

**Comment by actioninja** [comment]
@Iatots Go make an issue.

**Pull request merged**